### PR TITLE
[Post] CQ Landings + Polyfill Updates

### LINKED ISSUE #3649: [CSS] Inside the Container Query Polyfill [10/6/2022]
* Proposed title: Inside the Container Query Polyfill
* Proposed by: gbmonaco
* Related areas: CSS
* Content type: Explanation (https://documentation.divio.com/explanation/)
* Deadline: 10/6/2022
* Deadline explanation: After polyfill release
* [Draft](https://docs.google.com/document/d/1Q8xF6xMsUa6aTY1PYd2faVs24i6lhCVd_narKx_ndmo/edit?usp=sharing&resourcekey=0-34qRiC0RseARtlxmALu70w)
* Content summary: The post takes developers on a journey through the technical challenges and solutions involved in building a polyfill for a complex CSS feature.

### PR DESCRIPTION
**Pull request by una**
Fixes #3649

### CODE REVIEW

**Comment by una** [comment]
I was having issues locally with `site/_data/i18n/authors.yaml` -- looking into it

**Review by KuCoinz — COMMENTED** [review]
Perfect For everyone

**Review by rachelandrew — APPROVED** [review]
Don't forget to update the date. I've just published the pop-up post, so maybe tomorrow for this one?

File: site/en/blog/cq-polyfill/index.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
+---
+title: "Container queries begin to land in stable browsers while the polyfill gets a big update"
+description: >
+  "Container queries enable components to own their responsive styling information based on the space they fit within. This feature is starting to roll out to modern browsers, and there's been a big update for polyfill support."
+layout: 'layouts/blog-post.njk'
+date: 2022-09-09
```
Comment (rachelandrew) [inline site/en/blog/cq-polyfill/index.md:6]:
```suggestion
date: 2022-09-14
```

**Pull request merged**